docs/esp32/quickref.rst

```diff
@@ -422,9 +422,37 @@ See :ref:`machine.WDT <machine.WDT>`. ::
     from machine import WDT
 
     # enable the WDT with a timeout of 5s (1s is the minimum)
-    wdt = WDT(timeout=5000)
+    wdt = WDT(timeout=5000)  # timeout in milliseconds
     wdt.feed()
 
+
+When the WDT timeout is short, there is a software update issue:
+you don't have enough time to copy the updates to the device.
+ESP32 alows reinitialise the watchdog with a longer timeout – like an hour.
+
+More comprehansive example usage::
```

Comment:
A long example like this doesn't belong in the quickref.  The paragraph above is enough (to mention that the timeout can be reinitialised).